main.js

```diff
@@ -160,4 +160,152 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// ---------------- OPDRACHT 1: ARRAY METHODEN ----------------------
+
+// 1A
+const tvNames = inventory.map((tv) => {
+  return tv.name;
+});
+console.log(tvNames);
+
+//1B
+const tvSoldOut = inventory.filter((tv) => {
+  return tv.originalStock - tv.sold === 0;
+});
+console.log(tvSoldOut);
+
+//1C
+const tvAmbiLight = inventory.filter((tv) => {
+  return tv.options.ambiLight === true;
```

Comment:
Hier levert `tv.options.ambiLight` óók een boolean op. Daarom is ` === true` in principe overbodig:
```suggestion
  return tv.options.ambiLight;
```